imkar/scattering/analytical.py

```diff
@@ -0,0 +1,179 @@
+import numpy as np
+import pyfar as pf
+
+import numbers
+
+
+def rectangular(frequencies, phis, width, length, height, c=343):
+    """
+    This function computes the scattering coefficient of periodic rectangular
+    profiles, assuming that the extent is infinite.
+
+    This function is based on Ducourneau's method described by Prof. Embrechts
+    [#]_.
+
+    Parameters
+    ----------
+    frequencies : array, double
+        Vector of frequencies the scattering coefficient is computed at.
+    phis : array, double
+        Vector of incidences phi the scattering coefficient is computed at.
+        Values between 0°-90° allowed.
+    width : double
+        Width of the rectangulars in m.
+    length : double
+        Length of the regtangulars in m.
+    height : double
```

Comment:
```suggestion
    height : float
```
we use usaully float

Reply:
Changed